src/components/common/ColonyHome/partials/ReputationChart/ReputationChart.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from 'react';
+import { Id } from '@colony/colony-js';
+import clsx from 'clsx';
+
+import { defineMessages } from 'react-intl';
+import { useColonyContext } from '~hooks';
+import { notNull } from '~utils/arrays';
+import { formatText } from '~utils/intl';
+import WidgetBox from '~v5/common/WidgetBox';
+import DonutChart from '~v5/common/DonutChart';
+import TeamReputationSummaryRow from '~v5/common/TeamReputationSummary/partials/TeamReputationSummaryRow';
+import EmptyWidgetState from '~v5/common/WidgetBox/partials/EmptyWidgetState';
+import { useActionSidebarContext } from '~context/ActionSidebarContext';
+import { ACTION_TYPE_FIELD_NAME } from '~v5/common/ActionSidebar/consts';
+import { ACTION } from '~constants/actions';
+import { ChartData } from '~v5/common/DonutChart/types';
+
+import { getTeamReputationChartData } from './utils';
+import { summaryLegendColor } from './consts';
+
+const displayName = 'common.ColonyHome.ReputationChart';
+
+const MSG = defineMessages({
+  legendNoTeams: {
+    id: `${displayName}.legendNoTeams`,
+    defaultMessage: 'There are no teams with reputation in the Colony',
+  },
+  legendNoTeamsAction: {
+    id: `${displayName}.legendNoTeamsAction`,
+    defaultMessage: 'Create an action',
```

Comment:
This should say "Create a new action" just to keep in line with the other widget's calls to actions

![Screenshot from 2023-12-20 19-03-41](https://github.com/JoinColony/colonyCDapp/assets/1193222/a1573a26-1945-471d-995a-3a009a461fec)